src/test/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobViewTest.java

```diff
@@ -83,7 +86,7 @@ public void should_use_build_name_if_its_known() {
     public void should_know_the_url_of_the_last_build() {
         view = JobView.of(
                 a(job().whereTheLast(build().numberIs(22))),
-                locatedAt("job/project-name")
+            config, locatedAt("job/project-name")
```

Comment:
OK, so I refactored these new method params in without looking at the end result :) I'll tidy this oneto support one-liner test descriptions